### ISSUE TITLE
IO ult to allies

### ISSUE THREAD

**Issue by norms12** [ValveSoftware/Dota2-Gameplay#9921]
### Ability name

TETHER and RELOCATE

### Description

Players using IO to tether and relocate teammates to a deadspot where no one can skip. It is above the fountain of both base. 
Please fix this. 

### Example Match ID (and possibly Timestamp)

7219075711  whole game

**Comment by ThinkPortals** [comment]
You can disable help via the scoreboard (shared unit controls option list)

**Comment by BenLubar** [comment]
True, but Io should not be able to teleport out of bounds in the first place.
